sdk/src/main/scala/io/horizen/account/state/McAddrOwnershipMsgProcessor.scala

```diff
@@ -5,7 +5,7 @@ import io.horizen.account.abi.ABIUtil.{METHOD_ID_LENGTH, getABIMethodId, getArgu
 import io.horizen.account.fork.ZenDAOFork
 import io.horizen.account.proof.SignatureSecp256k1
 import io.horizen.account.state.McAddrOwnershipLinkedList._
```

Comment:
I would remove this import:
import io.horizen.account.state.McAddrOwnershipLinkedList._
now that we have more than one linkedlist, let's be sure we never access with a "default" import to one of them, to improve code readability

Reply:
Agree, done